srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -27,14 +27,11 @@ global
 
     chroot      /var/lib/haproxy
     pidfile     /var/run/haproxy.pid
-    maxconn     {{ (150 / salt['pillar.get']('haproxy:nbproc', 2)) | round(0, 'ceil') | int }}
+    maxconn     {{ salt['pillar.get']('haproxy:maxconn', 150) }}
```

Comment:
you need to divide the maxconn value by nbproc here. Does not mean anything now as nbproc value is 0, but moving ahead if this value changes then it will impact maxconn.